src/main/java/pl/krakow/PhoneBookApplication.java

```diff
@@ -2,6 +2,11 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import pl.krakow.domain.Customer;
+
+import java.util.ArrayList;
+import java.util.List;
 
 @SpringBootApplication
```

Comment:
Well, you were supposed NOT to use SpringBoot but use pure Spring.

Reply:
done!